### PR TITLE
Show stderr in childProcess.exec

### PR DESCRIPTION
**Pull request by Fatme**
Introduce a flag to show output from stderr
Add javac and gradle detection.

### CODE REVIEW

**Comment by justcodebuilduser** [comment]
:heart:

File: declarations.d.ts [hunk 1]
```diff
@@ -316,6 +320,10 @@ interface ISysInfoData {
 	monoVer: string;
 	/** git version string, as retunrnes by `git --version` **/
 	gitVer: string;
+	/** gradle version string as retunrnes by `gradle -v` **/
```
Comment (teobugslayer) [inline declarations.d.ts]: retunrnes -a spelling mistake. Not a merge stopper though

**Comment by teobugslayer** [comment]
:+1:

**Comment by justcodebuilduser** [comment]
:heart:

**Pull request merged**